protocol/0081-SUCM-successor_markets.md

```diff
@@ -70,7 +71,32 @@ Two proposals that name the same parent can be submitted. Both can be approved b
 
 A new market proposal sets parent market Id to a market that has settled. The parent market has non-zero insurance pool balance. If the new market clears the opening auction before `parent settlement time + market.value.windowLength` then the virtual stakes are carried over and the relevant fraction of the insurance pool is transferred over (<a name="0081-SUCM-006" href="#0081-SUCM-006">0081-SUCM-006</a>).
 
-A new market proposal sets parent market Id to a market that has settled. The parent market has non-zero insurance pool balance. If the new market clears the opening auction after `parent settlement time + market.value.windowLength` then no virtual stakes are carried over, there is no transfer into the insurance pool of the new market from the parent and the new market has no parent market Id set (<a name="0081-SUCM-007" href="#0081-SUCM-007">0081-SUCM-007</a>).
+A new market proposal sets parent market Id to a market that has settled. The parent market has non-zero insurance pool balance. If the new market clears the opening auction after `parent settlement time + market.value.windowLength` then no virtual stakes are carried over, there is no transfer into the insurance pool of the new market from the parent and the new market has no parent market Id set (<a name="0081-SUCM-007" href="#0081-SUCM-007">0081-SUCM-007</a>)
+
+Successor markets can't be enacted if the parent market is still in the "proposed" state (<a name="0081-SUCM-008" href="#0081-SUCM-008">0081-SUCM-008</a>)
+
+Successor markets can be enacted when the parent market is in opening auction (<a name="0081-SUCM-009" href="#0081-SUCM-009">0081-SUCM-009</a>)
+
+A successor market proposal can be enacted when the parent market is in one of the following states: Suspended, Active, Trading terminated or Settled (settled within the successor time window) (<a name="0081-SUCM-010" href="#0081-SUCM-010">0081-SUCM-010</a>)
+
+When a successor market is enacted, all other proposals (pending or proposed) are automatically rejected. Any LP submissions associated with these proposals are cancelled, and the funds are released (<a name="0081-SUCM-011" href="#0081-SUCM-011">0081-SUCM-011</a>)
```

Comment:
Can we please rephrase `all other proposals (pending or proposed) are automatically rejected`. It's simply not correct. There can be many proposals that have nothing to do with the successor or the market and those should be left alone. 

Proposals for a market that name the same parent should be rejected.